### ISSUE TITLE
10s blockchain request time out too small

### ISSUE THREAD

**Issue by guruvan** [dashpay/electrum-dash#18]
With a slow network connection (under 100kbps) 10s isn't enough time to receive a full chunk

L580 
   elif time.time() - req_time > 60:

(thought my servers had suddenly failed, but was my own network connections)

**Comment by guruvan** [comment]
not sure what the better timeout is - 30s seems also too short here, have it to 60 on my local host and testing

**Comment by guruvan** [comment]
yes - that's what I've done, exactly. have also messed with the SERVER_RETRY_INTERVAL 
the longest I've seen the "blockchain request timeout" message take to appear is 37s, not sure if I did that or not. ;) 

meanwhile - if you could have a quick test of the following servers:
electrum.maza.club:50002:s
52.36.21.221:50001:t
52.37.252.153:50001:t

I'm attempting to install a remote desktop for testing as well.

**Comment by Kefkius** [comment]
@guruvan I can connect to all three of those.

`SERVER_RETRY_INTERVAL` is used as follows:

If we're not connected to any server and `auto_connect` is not enabled, and we have disconnected from `default_server`, then after `SERVER_RETRY_INTERVAL` we try to connect to `default_server` again.

You may want to try removing `~/.electrum-dash/blockchain_headers`?

**Comment by guruvan** [comment]
Each time I test I remove the recent_servers, edit config to correct server, and remove blockchain_headers - should be a clean test. All I'm not doing is changing the wallet out. 

The client definitely retries - it keeps retrying, in fact, which is how I've noticed it's not transient, or some startup issue. Simply, my network today is slowwww. But I'd think that electrum should work over a slow connection. (It will, but apparently it won't get a whole chunk, much less hundreds of 'em  to get started up)

**Comment by guruvan** [comment]
also thanks for testing that :)
